mesheryctl/pkg/utils/error.go

```diff
@@ -42,6 +42,7 @@ var (
 	ErrInvalidArgumentCode    = "mesheryctl-1118"
 	ErrGeneratingIconsCode    = "mesheryctl-1119"
 	ErrClearLineCode          = "mesheryctl-1120"
+	ErrConnectionRefusedCode  = "mesheryctl-1121"
```

Comment:
next available code is available https://github.com/meshery/meshery/blob/5a67d1ee1916d8cc14e72221b86cd95eac018572/mesheryctl/helpers/component_info.json#L4, you will need to take the current `next_error_code` value then modify it (increment +1)

Reply:
updated